api/v1beta1/conditions.go

```diff
@@ -0,0 +1,123 @@
+	KeystoneServiceOSEndpointsReadyCondition condition.Type = "KeystoneServiceOSEndpointsReady"
+
+	// KeystoneServiceOSUserReadyCondition Status=True condition which indicates if the service user got created in the keystone instance is ready/was successful
+	KeystoneServiceOSUserReadyCondition condition.Type = "KeystoneServiceOSUserReady"
+)
+
+//
+// Keystone Reasons used by API objects.
+//
+const ()
+
+//
+// Common Messages used by API objects.
+//
+const (
+
+	//
+	// KeystoneAPIReady condition messages
+	//
+	// KeystoneAPIReadyInitMessage
+	KeystoneAPIReadyInitMessage = "KeystoneAPI not started"
+
+	// KeystoneAPIReadyMessage
+	KeystoneAPIReadyMessage = "KeystoneAPI ready"
+
+	// KeystoneAPIReadyNotFoundMessage
+	KeystoneAPIReadyNotFoundMessage = "KeystoneAPI not found"
+
+	// KeystoneAPIReadyWaitingMessage
+	KeystoneAPIReadyWaitingMessage = "KeystoneAPI not yet ready"
+
+	// KeystoneAPIReadyErrorMessage
+	KeystoneAPIReadyErrorMessage = "KeystoneAPI error occured %s"
+
+	//
+	// AdminServiceClientReady condition messages
+	//
+	// AdminServiceClientReadyInitMessage
+	AdminServiceClientReadyInitMessage = "Admin client not started"
+
+	// AdminServiceClientReadyMessage
+	AdminServiceClientReadyMessage = "Admin client ready"
+
+	// AdminServiceClientReadyWaitingMessage
+	AdminServiceClientReadyWaitingMessage = "Admin client not yet ready"
+
+	// AdminServiceClientReadyErrorMessage
+	AdminServiceClientReadyErrorMessage = "Admin client error occured %s"
+
+	//
+	// KeystoneServiceOSServiceReady condition messages
+	//
+	// KeystoneServiceOSServiceReadyInitMessage
+	KeystoneServiceOSServiceReadyInitMessage = "Keystone Service registration not started"
+
+	// KeystoneServiceOSServiceReadyMessage
+	KeystoneServiceOSServiceReadyMessage = "Keystone Service %s - %s ready"
+
+	// AdminServiceClientReadyErrorMessage
+	KeystoneServiceOSServiceReadyErrorMessage = "Keystone Service error occured %s"
+
+	//
+	// KeystoneServiceOSEndpointsReady condition messages
+	//
+	// KeystoneServiceOSEndpointsReadyInitMessage
+	KeystoneServiceOSEndpointsReadyInitMessage = "Keystone Endpoints registration not started"
+
+	// KeystoneServiceOSEndpointsReadyMessage
+	KeystoneServiceOSEndpointsReadyMessage = "Keystone Endpoints %+v ready"
+
+	// KeystoneServiceOSEndpointsReadyErrorMessage
+	KeystoneServiceOSEndpointsReadyErrorMessage = "Keystone Endpoints error occured %s"
+
+	//
+	// KeystoneServiceOSUserReady condition messages
+	//
+	// KeystoneServiceOSUserReadyInitMessage
+	KeystoneServiceOSUserReadyInitMessage = "Keystone Service user registration not started"
+
+	// KeystoneServiceOSUserReadyMessage
+	KeystoneServiceOSUserReadyMessage = "Keystone Service user %s ready"
+
+	// KeystoneServiceOSUserReadyWaitingMessage
+	KeystoneServiceOSUserReadyWaitingMessage = "Keystone Service user not yet ready"
+
+	// KeystoneServiceOSUserReadyErrorMessage
+	KeystoneServiceOSUserReadyErrorMessage = "Keystone Endpoints error occured %s"
```

Comment:
The value here looks like it should say something different.  Refer to service user instead of endpoints?

Reply:
right, good catch. will update

Reply:
done